src/cli.js

```diff
@@ -0,0 +1,74 @@
+#! /usr/bin/env node
+
+'use strict'
+
+const YargsPromise = require('yargs-promise')
+const yargs = require('yargs')
+const process = require('process')
+const log = require('debug')('repo-migrations:cli')
+
+const commands = require('./commands')
+
+function print (msg, newline) {
+  if (newline === undefined) {
+    newline = true
+  }
+
+  if (msg === undefined) {
+    msg = ''
+  }
+  msg = newline ? msg + '\n' : msg
+  process.stdout.write(msg)
+}
+
+async function main (args) {
+  const cli = yargs()
+    .option('repo-path', {
+      desc: 'Path to the IPFS repo',
+      type: 'string'
+    })
+    .option('migrations', {
+      desc: 'Path to folder with migrations. Default: bundled migrations',
+      type: 'string'
+    })
+    .command(commands.migrate)
+    .command(commands.status)
+    .command(commands.add)
+    .demandCommand(1, 'You need at least one command before continuing')
+    .strict()
+    .fail((msg, err, yargs) => {
+      if (err) {
+        throw err // preserve stack
+      }
+
+      if (args.length > 0) {
```

Comment:
Why only print the message `if (args.length > 0)`?

Reply:
This is to suppress message `You need at least one command before continuing` when you run the command without any parameters that display the help page.

I took most of this file from `js-ipfs` I think.

Reply:
Makes sense 👍 
Please add a comment to the code explaining that